scripts/fetch_and_build_deb

```diff
@@ -175,4 +175,18 @@ mjobs="$(expr $procs + 1)"
 
 DEB_BUILD_OPTIONS="parallel=${mjobs}" debuild --prepend-path /usr/local/bin --preserve-envvar CONF_EXTRA_VERSION --preserve-envvar UNENCRYPTED_PACKAGE --preserve-envvar PACKAGE_ENCRYPTION_KEY -uc -us -B --lintian-opts --profile debian --allow-root
 
+## Rename all *.ddeb files into *.deb since we upload debug packages as deb
+for file in ../*.ddeb
+do
+  # If check below is to prevent error, when there is not ddeb file exists
+  # In this case file nane becomes '../*.ddeb'
+  if [ ${file} != "../*.ddeb" ]; then
+    echo "Renaming ${file} into ${file%.ddeb}.deb"
+    mv "$file" "${file%.ddeb}.deb";
+  else
+    echo "There are no ddeb files to rename."
+  fi
+done
+#----------
```

Comment:
Instead of skipping debug packages for Debian releases, can we make sure debug packages are created by using output of `lsb_release --id` ?:

```suggestion
release_id = lsb_release --id
if release_id contains "Debian":
  # .ddeb suffix is used when naming debug packages for Debian releases.
  # For this reason, we rename all *.ddeb files to *.deb since we upload
  # debug packages with .deb suffix.
  for file in ../*.ddeb
  do
    echo "Renaming ${file} to ${file%.ddeb}.deb"
    mv "$file" "${file%.ddeb}.deb";
  done
```

I don't think CI should succeed if debug packages are not available for Debian releases.
If we silently allow not building debug packages for Debian releases, then we can't make sure that we have necessary debug packages for all distros.

Reply:
@onurctirtir Debug packages creation is determined in debian rules file. In case of a problem such as rule definition changes, we can not create normal packages which may cause problems when urgent packages are needed. Debug packages are important but not killer. In such case, we will have big problems. I think we should keep as it is not to cause such problems

I can not understand the lsb_release comment. Debian releases ( buster stretch) already have debug packages. Why do we need another check?

Reply:
Discussed in person and realized that we were anyway not verifying if debug packages are created or not.
So just leaving some minor suggestions regarding code comments etc.:

```suggestion
## Rename all *.ddeb files to *.deb since we upload debug packages with .deb suffix
for filename in ../*.ddeb
do
  # .ddeb suffix is automatically chosen when naming debug packages for Ubuntu
  # releases. For this reason, we rename all *.ddeb files to *.deb since we upload
  # debug packages with .deb suffix.
  if [ ${filename} != "../*.ddeb" ]; then
    echo "Renaming ${file} to ${file%.ddeb}.deb"
    mv "$filename" "${filename%.ddeb}.deb";
  else
    # If there is no files with .ddeb suffix, then "filename" here will exactly be
    # equal to our search pattern, i.e. '../*.ddeb'.
    echo "There are no *ddeb files to rename."
  fi
done
```